code/processes/tailer.q

```diff
@@ -46,9 +47,28 @@ getprocs:{[x;y]
         }
 
 .servers.register[.servers.procstab;.tailer.tailreadertypes;1b]
+.servers.register[.servers.procstab;.tailer.tailsorttypes;1b]
 
 \d .
 
+/- eod - send end of day message to main tailsort process
+endofday:{[pt;processdata]
+  .lg.o[`eod;"end of day message received - ",spt:string pt];
+  /- call datastripeendofday
+  .wdb.datastripeendofday[pt;processdata];
+  /- find handle to send message to tailsort process
+  ts:exec w from .servers.getservers[`proctype;.tailer.tailsorttypes;()!();1b;0b];
+  /- exit early if no tailsort process connected
+  if[0=count ts;.lg.e[`connection;"no connection to the ",(string .tailer.tailsorttypes)," could be established, failed to send end of day message"];:()];
```

Comment:
what happens if we don't have a tailsort process? I know this line will throw an error and return early, but what state will the data be left in? will we still be able to query the data for this day etc.?

Reply:
Might be worth having a discussion on this point, and also looking at what standard TorQ WDB does in this scenario

Reply:
I've created an endofdaysort function in the tailer to be called if the tailsort process isn't connected - it uses functions from the tailsort.q script and I've tried using a sleep command to avoid multiple tailers trying to write to the HDB at the same time